kview/src/main/java/dev/ikm/komet/kview/mvvm/view/navigation/PatternNavEntryController.java

```diff
@@ -255,6 +246,7 @@ protected void updateItem(Object item, boolean empty) {
 
                         setContentDisplay(ContentDisplay.GRAPHIC_ONLY);
                         label.setText(entityDescriptionText);
+                        Tooltip.install(label, new Tooltip(entityDescriptionText));
```

Comment:
it's best to create this tooltip outside of the updateItem method. This updateItem method gets called everytime a cell gets updates with a new value. So you'll be creating lots of Tooltip objects this way.